packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -239,11 +269,16 @@ export function registerLocalNetwork(
 }
 
 export function isNetwork(chainId: ChainId) {
+  const customChains = getCustomChainsFromLocalStorage().filter(
+    chain => chain.chainID === chainId
+  )
```

Comment:
this reads like you're only looking for the custom chain that matches with the `chainId` param passed into the function

so i'd expect the const to be `customChain` of type `ChainWithRpcUrl` rather than `customChains` of type `ChainWithRpcUrl[]`

use `.find`?

Reply:
done